packages/govuk-frontend/src/govuk/components/input/input.yaml

```
@@ -30,12 +30,12 @@ params:
   - name: label
     type: object
     required: true
-    description: Options for the label component.
+    description: Additional options for the label used by the text input component.
```

Again, having a `label.text` or `label.html` is pretty important, so unsure about 'Additional'.

---

Good spot, this one shouldn't say "Additional…"

```suggestion
    description: Can be used to add a label to the text input component.
```

---

On second thoughts, this one has `required: true`

For mandatory fields we've added "Additional options…" to show it was in addition to the defaults

Linking back to https://github.com/alphagov/govuk-frontend/pull/4450#discussion_r1397583810 where it was also confusing

---

I've dropped the entire "Additional options for…" as we've done in https://github.com/alphagov/govuk-frontend/pull/4450#discussion_r1401799402

```yaml
    description: The label used by the text input component.
```